src/applications/MachineShopSimulator.java

```diff
@@ -194,4 +150,28 @@ public static void main(String[] args) {
     public int getTimeNow() {
         return timeNow;
     }
+
+    public EventList geteList() {
+        return eList;
+    }
+
+    public void seteList(EventList eList) {
+        this.eList = eList;
+    }
+
+    public Machine[] getMachine() {
+        return machine;
+    }
+
+    public void setMachine(Machine[] machine) {
+        this.machine = machine;
+    }
+
+    public int getLargeTime() {
+        return largeTime;
+    }
+
+    public void setLargeTime(int largeTime) {
+        this.largeTime = largeTime;

```

Comment:
This looks like inappropriate intimacy to me.  Aren't all of these private, or at least implicitly private?